templates/shortcode-receipt.php

```diff
@@ -84,8 +84,20 @@
 				<ul class="edd_receipt_fees">
 				<?php foreach ( $order->get_fees() as $fee ) : ?>
 					<li>
-						<?php if ( ! empty( $fee->description ) ) : ?>
-							<span class="edd_fee_label"><?php echo esc_html( $fee->description ); ?></span>
+						<?php if ( 'order_item' === $fee->object_type || ! empty( $fee->description ) ) : ?>
+							<span class="edd_fee_label">
+								<?php
+								if ( 'order_item' === $fee->object_type ) {
+									$order_item = edd_get_order_item( $fee->object_id );
+									if ( $order_item ) {
+										echo esc_html( $order_item->product_name ) . ':&nbsp;';
```

Comment:
This is actually giving me a double space. I don't think the `&nsbp;` is needed.

![Screenshot from 2021-02-10 14-36-39](https://user-images.githubusercontent.com/6324272/107524181-6bfa1d80-6bad-11eb-9cb3-5316c24b3a0a.png)